### PR TITLE
Add idea class and queryselectors

### PR DESCRIPTION
**Pull request by DrewBradley**
#### What does  this PR do?
Class constructor for instantiating a new idea card was added, along with querySelectors for the elements that will be manipulated in the DOM.

#### Where should the reviewer start?
The reviewer should start by confirming the necessary properties in the Idea class constructor.

#### How should this be manually tested?
This can be tested by logging a new instantiation of the idea class to the console.

#### Any background context you want to provide?
This is the main constructor that will be used for this application.

### CODE REVIEW

File: src/idea.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Idea {
+  constructor(title, body){
+    this.id = Date.now();
+    this.title = title;
+    this.body = body;
+    this.star = false;
+  }
+  saveToStorage(){
+
```
Comment (JeffKersting) [inline src/idea.js:9]: Research local storage, and implement here

File: src/idea.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Idea {
+  constructor(title, body){
+    this.id = Date.now();
+    this.title = title;
+    this.body = body;
+    this.star = false;
+  }
+  saveToStorage(){
```
Comment (DrewBradley) [inline src/idea.js:8]: Thank you for adding the shells of the necessary methods that we will be using later on.

File: src/main.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Idea from './idea.js';
+
+var showStarredButton = document.querySelector(".show-favorite");
```
Comment (DrewBradley) [inline src/main.js:3]: Are we certain we need all these querySelectors?

File: src/idea.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+class Idea {
+  constructor(title, body){
+    this.id = Date.now();
```
Comment (colefiscus) [inline src/idea.js:3]: Something I'm just curious about - is there another way to create a unique identifier than Date.now()?

**Review by colefiscus — APPROVED** [review]

**Pull request merged**